modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/array.ts

```diff
@@ -30,9 +30,16 @@ export function filterArrayByQuery<T>(
 			const localizedValue = ((item as {[key: string]: unknown}) as {
 				[key: string]: LocalizedValue<string>;
 			})[str];
-			const label = (localizedValue as LocalizedValue<string>)[
-				defaultLanguageId
-			] as string;
+
+			const localizedLabels = localizedValue as LocalizedValue<string>;
+
```

Comment:
Hey @naraandrade, I think we can use `localizedValue` right below instead of creating this `const`, what do you think?

Reply:
Hi @danvb112, I created this const to avoid repeating the type assertion. Without the const the code would be:

                    let label = (localizedValue as LocalizedValue<string>)[defaultLanguageId] as string;

			if (!label) {
				label = (localizedValue as LocalizedValue<string>)[
					item.defaultLanguageId as Locale
				] as string;
			}

Reply:
Hmm it's ok for me then :smile: